src/main/java/seedu/address/model/student/Remark.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.student;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Remark in the address book.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidRemarkName(String)}
+ */
+public class Remark {
```

Comment:
Good practice of encapsulation of Remark and keeping the same abstraction level for the Student fields